package.json

```diff
@@ -49,7 +49,7 @@
     "eslint-plugin-mocha": "^9.0.0",
     "ethereum-waffle": "^3.4.4",
     "ffjavascript": "^0.2.57",
-    "hardhat": "^2.13.0",
+    "hardhat": "^2.17.0",
```

Comment:
to provide some context, this is needed b/c of https://github.com/NomicFoundation/hardhat/issues/4164